llama_stack/providers/utils/memory/openai_vector_store_mixin.py

```diff
@@ -614,7 +614,7 @@ async def openai_attach_file_to_vector_store(
                 )
                 vector_store_file_object.status = "completed"
         except Exception as e:
-            logger.error(f"Error attaching file to vector store: {e}")
+            logger.exception("Error attaching file to vector store")
```

Comment:
this try-catch block is terrible because it just wraps a large amount of code each line of which can produce different _legitimate_ errors (not just unknown things). so just a `logger.error()` ain't good enough. I will clean that up in a follow-up.